debian/salsa-ci.yml

```diff
@@ -46,10 +46,9 @@ build:
     - ccache -s # Show ccache stats to validate it worked
     - mv ${CCACHE_TMP_DIR} ${CCACHE_WORK_DIR}
 
-build bullseye-backports:
-  extends: .build-package
-  variables:
-    RELEASE: bullseye-backports
+# build jobs often needs more than default GitLab timeout (1h)
+.build-package:
+  timeout: 3h
 
```

Comment:
Should Gitlab runtime timeout used as I don't know does this even work?

Reply:
https://salsa.debian.org/mariadb-team/mariadb-server/-/commit/7abb3a4f813ae6032dc9bcce40d0a7c00df6c50d
  GitLab CI now supports timeout for specific jobs